colournaming/templates/index.html

```diff
@@ -169,26 +168,8 @@ <h2 id="research">
             </ul>
         </div>
         <div class="padded">
-            <h3>Purpose of the research:</h3>
-            <ul>
-                <li>To determine multicultural sets of basic colour terms, compounds, modifiers and secondary colour names which are in wide use.</li>
-                <li>To locate multicultural colour names in Munsell, sRGB, CIELAB and CIECAM02.</li>
-                <li>To investigate colour naming and colour categorization within different cultures.</li>
-                <li>To develop a colour-naming model.</li>
-            </ul>
-        </div>
-        <div class="padded">
-            <h3>What you will do in this <a href="http://colornaming.net">experiment</a> (approximate duration 10 min):</h3>
-            <ul>
-                <li>We will ask you to give us information on your display technology and viewing conditions.</li>
-                <li>We will test your colour vision with a 90 sec short video.</li>
-                <li>We will present to you a series of colour patches, one sample at a time, and ask you to name it by using natural language.</li>
-                <li>We will ask you to give us information on your cultural background.</li>
-            </ul>
-        </div>
-        <div class="padded">
             <h3>Crowdfunding Project:</h3>
-            <p>In September 2016, we've run a successful Kickstarter project, entitled <a href="https://www.kickstarter.com/projects/111984715/colours-of-babel">Colours of Babel</a>, to upgrade our online experiment to run on all new devises, minimize security threats and maintain our online presence free without advertisements. We thank all our 96 supporters for helping us raise 112% of our funding goal. You can find below a colour-coded list with our most precious donors:</p>
+            <p>In September 2016, we run a successful Kickstarter project, entitled <a href="https://www.kickstarter.com/projects/111984715/colours-of-babel">Colours of Babel</a>, to upgrade our online experiment to run on all new devises, minimize security threats and maintain our online presence free without advertisements. We thank all our 96 supporters for helping us raise 112% of our funding goal. You can find below a colour-coded list with our most precious donors:</p>
```

Comment:
Couple of typos:
"we run a successful Kickstarter" should be "we ran a successful Kickstarter"
"devises" should be "devices"